projects/index.html

```diff
@@ -47,14 +47,57 @@
 				<div class="mdl-cell mdl-cell--2-col mdl-cell--hide-tablet mdl-cell--hide-phone"></div>
 				<div class="demo-content mdl-color--white mdl-shadow--4dp content mdl-color-text--grey-800 mdl-cell mdl-cell--8-col">
 					<h3>Projects</h3>
-					<div class="demo-card-wide mdl-card mdl-shadow--2dp">
-						<div class=mdl-card__title style="background-color: grey">
-							<h2 class=mdl-card__title-text>Omeclone</h2>
+					<a href="https://paper.webdevlabs.in/" style="text-decoration: none;">
+						<div class="demo-card-wide mdl-card mdl-shadow--2dp">
+							<div class="mdl-card__supporting-text">
+								<h2 class=mdl-card__title-text>Paper-Extension</h2>
+							</div>
+							<div class=mdl-card__title style="background: url(../images/paperext.png) center; background-size: cover;">
+							</div>
+							<div class=mdl-card__supporting-text>
+								A chrome extension to sync the highlighted text from web to a dropbox paper.
```

Comment:
Can this be "A chrome extension to save selected text from a webpage to dropbox paper"? I believe that makes it more clear.

Reply:
sounds good!